packages/devtools/lib/src/vm_service_wrapper.dart

```diff
@@ -186,10 +186,10 @@ class VmServiceWrapper implements VmService {
     );
   }
 
+  @deprecated
   @override
   Future<CpuProfile> getCpuProfile(String isolateId, String tags) {
-    return _trackFuture(
-        'getCpuProfile', _vmService.getCpuProfile(isolateId, tags));
+    throw UnsupportedError('Private service RPC');
```

Comment:
I think we're calling this method (`getCpuProfile()`) in the app currently. Perhaps change to using `callMethod()` here?

Reply:
We're actually calling `getCpuProfileTimeline` in `getCpuProfile` in `CpuProfilerService`'s. I was confused at first as well, but the analyzer doesn't seem to think this is used anywhere else (gave me a deprecation hint when I originally tried to deprecate `clearCpuProfile`).